storage-api/pom.xml

```diff
@@ -113,14 +113,13 @@
       <artifactId>slf4j-api</artifactId>
       <version>${slf4j.version}</version>
     </dependency>
-
-    <!-- test inter-project -->
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
       <version>${guava.version}</version>
-      <scope>test</scope>
```

Comment:
Hi, All. This seems to cause a regression at `hive-storage-api` 2.8.0.
- https://mvnrepository.com/artifact/org.apache.hive/hive-storage-api/2.8.0